### ISSUE TITLE
Can't boot post installation 

### ISSUE THREAD

**Issue by hMounir** [Xigtun/xps-9570-mojave#32]
Hi, i need help with my installation, I have able to install Mojave normally, but after installing, it makes a reboot right after showing the apple logo, i don't know why this happens. I used the clover install folder

**Comment by genie137** [comment]
Have you found a fix for this? I'm running in to the same issue.

**Comment by Raymonf** [comment]
Did you swap your Clover folder after installation? @genie137

**Comment by hMounir** [comment]
no, I didn't, I can't even boot post installation for me to swap it
